File: 30 Days of Code/Day 16 Exceptions - String to Integer/Exception.java
import java.util.Scanner;

class Exception {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		String str = input.nextLine();
		Integer number = 0;

		try {
			number = Integer.parseInt(str);
			System.out.println(number);
		} catch (NumberFormatException e) {
			System.out.println("Bad String");
		}
	}
}

File: 30 Days of Code/Day 16 Exceptions - String to Integer/exception.cpp
#include <iostream>
#include <string.h>
using namespace std;

int main()
{
	string str = "";
	cin >> str;
	int number = 0;

	try {
		number = stoi(str);
		cout << number;
	} catch (...) {
		cout << "Bad String";
	}
}
		
